pkg/agent/util/iputils/iputils.go

```diff
@@ -0,0 +1,148 @@
+// Copyright 2020 Antrea Authors
```

Comment:
Since iputils is under util, maybe just `util/ip/`.
And given that the package is also used in antctl and perhaps controller, maybe move it to `pkg/util/ip/`.

Reply:
Yes, I originally thought move it out of pkg/agent, and since util is not there I was not sure. I moved it now as per your review.